CHANGELOG.md

```diff
@@ -1,3 +1,119 @@
+## v0.7.0 - 2021-03-01
+
+We're happy to announce the release of Lokomotive v0.7.0 (Ghan).
+
+### Changes in v0.7.0
+
+#### Kubernetes updates
+
+- Update Kubernetes to v1.20.4 ([#1410](https://github.com/kinvolk/lokomotive/pull/1410)).
+
+#### New components
+
+- Add component node-problem-detector ([#1384](https://github.com/kinvolk/lokomotive/pull/1384)).
+
+#### Component updates
+
+- Update `aws-ebs-csi-driver` from `0.7.0` to `0.9.0` ([#1393](https://github.com/kinvolk/lokomotive/pull/1393)).
+
+- Update `web-ui` to `0.2.1` ([#1412](https://github.com/kinvolk/lokomotive/pull/1412)).
```

Comment:
We should write version update in same format as above i.e `from ... to ...`

Reply:
Done.